lib/active_model/serializer.rb

```diff
@@ -154,6 +154,16 @@ def links
       self.class._links
     end
 
+    # @api private
+    # Returns an ActiveModel::Name for the serializer's object
+    def model_name
```

Comment:
btw, have you seen https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model_serializers/model.rb and that supporting `[ActiveModel::Errors](https://github.com/rails/rails/blob/4-2-stable/activemodel/lib/active_model/errors.rb)` (ref #1004) would require the model extend `ActiveModel::Name`?  thoughts on that (after adding tests to show the effect of this change)?